probert/tests/test_multipath.py

```diff
@@ -0,0 +1,76 @@
+import mock
+import subprocess
+
+from probert import multipath
+from probert.tests.helpers import CiTestCase
+
+
+class TestMultipath(CiTestCase):
+
+    @mock.patch('probert.multipath.subprocess.run')
+    def test_multipath_show_paths(self, m_run):
+        mp_out = " ".join([self.random_string() for x in range(0, 8)])
+        cp = subprocess.CompletedProcess(args = ['foo'], returncode = 0,
+                                         stdout = mp_out.encode('utf-8'),
+                                         stderr = "")
+        m_run.return_value = cp
+        expected_result = [multipath.MPath(*mp_out.split())._asdict()]
+        result = multipath.multipath_show_paths()
+        self.assertEqual(expected_result, result)
+
+    @mock.patch('probert.multipath.subprocess.run')
+    def test_multipath_show_paths_skips_unparsable_output(self, m_run):
+        lines = ["1 2 3 4 5 6 7 8", "", "a b c d e f g h"]
+        mp_out = "\n".join(lines)
+        cp = subprocess.CompletedProcess(args = ['foo'], returncode = 0,
```

Comment:
`subprocess.CompletedProcess` was introduced in 3.5; does probert only target Ubuntu systems that are xenial or later?

Reply:
Yes, we're bionic+ only